test/Dialect/Calyx/emit.mlir

```diff
@@ -331,3 +331,52 @@ module attributes {calyx.entrypoint = "main"} {
     }
   } {toplevel}
 }
+
+
+// -----
+
+module attributes {calyx.entrypoint = "main"} {
+  // CHECK: import "primitives/float/mulFN.futil";
+  calyx.component @main(%in0: f32, %clk: i1 {clk}, %reset: i1 {reset}, %go: i1 {go}) -> (%out0: f32, %done: i1 {done}) {
+    // CHECK: std_ieee754.mul_0 = std_ieee754.mul(8, 24, 32);
```

Comment:
This check command looks wrong. In native Calyx, the name cannot contain `.` in it...